src/esp/assets/ResourceManager.cpp

```diff
@@ -2056,7 +2080,14 @@ void ResourceManager::addComponent(
   // Add a drawable if the object has a mesh and the mesh is loaded
   if (meshIDLocal != ID_UNDEFINED) {
     const int meshID = metaData.meshIndex.first + meshIDLocal;
-    Magnum::GL::Mesh& mesh = *meshes_.at(meshID)->getMagnumGLMesh();
+    Magnum::GL::Mesh* mesh = meshes_.at(meshID)->getMagnumGLMesh();
+    if (getCreateRenderer()) {
+      CORRADE_ASSERT(mesh, "::addComponent : GL mesh expected but not found", );
```

Comment:
"::addComponent :" is incomplete. 
You need:
"ClassName::functionName()"
Please fix this problem in this PR (a couple of places needs to be fixed.)